lib/uvobot.rb

```diff
@@ -8,19 +8,19 @@ def run(release_date)
     if @scraper.issue_ready?(release_date)
       notify_announcements(release_date)
     else
-      @notifier.new_issue_not_published
+      @notifiers.each { |n| n.send(:new_issue_not_published) }
     end
   end
 
   private
 
   def notify_announcements(release_date)
-    page_info, announcements = @scraper.get_announcements(release_date)
+    page_info, announcements = @scraper.get_full_announcements(release_date)
```

Comment:
Podla mna vsetko info co na stiahnutie detailu potrebujes sa uz posiela. Mas tam v `matching_announcements_found` parameter `announcements` kde je url detailu - z toho to vies vsetko potiahnut. Nie?

Violation SRP nevidim.